app/core/Engine/controllers/AccountsController/logger.ts

```
@@ -0,0 +1,19 @@
+import { AccountsControllerState } from '@metamask/accounts-controller';
+import Logger from '../../../../util/Logger';
+import { defaultAccountsControllerState } from './utils';
+
+export function logAccountsControllerCreation(
+  initialState?: AccountsControllerState,
+) {
+  if (!initialState) {
+    Logger.log('Creating AccountsController with default state', {
+      defaultState: defaultAccountsControllerState,
+    });
+  } else {
+    Logger.log('Creating AccountsController with provided initial state', {
```

@sethkfman will using Logger.log ensure these logs show up in sentry?

---

They appear as breadcrumbs, which are attached to traces if an error occurs

---

@Cal-L if we want to guarantee that a log gets sent to sentry should we use log.error? or is that a misuse of this pattern?

---

@owencraston Yes, log.error is what you would want to make sure it is logged. The breadcrumbs are helpful if you log and error and wanted to see what occurred before that error.

---

After some consideration I don't think we want to report these as errors since it all add extra burden on the platform team to triage. Instead these will be added as breadcrumbs to crashes which is really what we care about.